lib/metrics.js

```diff
@@ -156,6 +162,7 @@ function printMetrics({raw = false, format = 'console', methods} = {}) {
     });
 
     const methodNames = _.isArray(methods) ? methods : _.keys(summaries);
+    const timeOrigin = performance.timeOrigin;
```

Comment:
Not super clear why we need to worry about `timeOrigin` here. I recall reading something about this in the docs but maybe we can leave a mention somewhere. e.g. why do we only subtract the `timeOrigin` from a "start time" but not a "duration" with some brief explanation might be valuable?

Reply:
`timeOrigin` replaces `PERFORMANCE_OFFSET`: https://github.com/Expensify/react-native-onyx/pull/101/files#diff-d5767fc7b9c776a9683e5059ef6f67713b836b8dbf6781461ae950228a478aa1L14

`react-native-performance` "suffers" from the same issue that made `PERFORMANCE_OFFSET` necessary
the `startTime` of the marks would not be relative to app launch, but relative to system uptime 
In order to see a `startTime` (or `performance.now()`) relative to app launch we need to subtract `timeOrigin` 
I'll add a comment about it

Reply:
Sounds good, thanks!